examples/update-demo/4-rolling-update.sh

```diff
@@ -26,8 +26,8 @@ fi
 NEW_IMAGE=${1:-kitten}
 TIMING=${2:-10s}
 export KUBE_REPO_ROOT=${KUBE_REPO_ROOT-$(dirname $0)/../..}
-export KUBECFG=${KUBECFG-$KUBE_REPO_ROOT/cluster/kubecfg.sh}
+export KUBECTL=${KUBECTL-$KUBE_REPO_ROOT/cluster/kubectl.sh}
 
-set -x
+SCHEMA=${KUBE_REPO_ROOT}/examples/update-demo/kitten-rc.yaml
 
-$KUBECFG -image $DOCKER_HUB_USER/update-demo:$NEW_IMAGE -u $TIMING rollingupdate update-demo
+cat ${SCHEMA} | sed "s/DOCKER_HUB_USER/${DOCKER_HUB_USER}/" | ${KUBECTL} rollingupdate update-demo-nautilus -f -
```

Comment:
It would be nice to be able to run these through the examples test:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/examples/examples_test.go#L100

But the substitution makes it not work. I have a couple ideas about how to fix that, but it doesn't need to be done in this PR.

Please add a todo.